crowbar_framework/config/routes.rb

```diff
@@ -122,6 +123,7 @@
   post "crowbar/:barclamp/1.0/proposals/commit/:id(.:format)", action: "proposal_commit", controller: "barclamp"
   get "crowbar/:barclamp/1.0/proposals/status(.:format)", action: "proposal_status", controller: "barclamp"
   delete "crowbar/:barclamp/1.0/proposals/:id(.:format)", action: "proposal_delete", controller: "barclamp"
+  post "crowbar/:barclamp/1.0/proposals/reset/:id(.:format)", action: "proposal_reset", controller: "barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [110/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)